frontEnd/src/components/filters/filters.jsx

```diff
@@ -26,13 +27,13 @@ function Filters({hackathon}) {
              [e.target.name]: e.target.value,
          });
      };
-     useEffect(() => {
+      useEffect(() => {
          let filteredHackathons = hackathon;
 
          // Filter by theme
          themeFilter !== ''
              ? (filteredHackathons = filteredHackathons.filter((hackathon) =>
-                   hackathon.tags.includes(themeFilter.toLowerCase())
+                   hackathon.Themes[0].includes(themeFilter.toLowerCase())
```

Comment:
Having `Themes[0]` does not follow your data model. Please check for logical errors while saving data in `Themes`. It seems like your logic is saving all the themes as comma separated values at `0th` index of the `Themes` array

Reply:
Please update this according to new changes